BLSQ/iaso
# IA-3482 IA-1694 IA-1728:  popups + remove Redux

Orgunit popup and instance popup are acting weird while using clustering.

Those two component are the last one's to really use redux store. We can finally remove all redux related code


Related JIRA tickets : IA-3482   + IA-1694 + IA-1728


## Self proofreading checklist


## Changes

- fix popups to use their own hook and fetch details infos
- custom hook to know if popup is open or not on map and trigger the call
- remove instances reducer, action
- remove org unit reducer and action
- remove all redux dependencies
- fix js test using redux

## How to test

Make sure you have two org units and two instances with the same location.
Open org unit search and display those two org units on the map, click on it, you should be able to open the popup with org units infos
Make the same thing with instances.

Play with org units list, detail, instances list, details


## Print screen / video


https://github.com/user-attachments/assets/90638959-9b20-4f1d-aa3c-76dfbbebadac



## Notes

Cypress tests are passing here : https://github.com/BLSQ/iaso/actions/runs/11405440785/job/31736948494
